scheduler/src/cook/tools.clj

```diff
@@ -843,10 +843,9 @@
 
 (defn below-quota?
   "Returns true if the usage is below quota-constraints on all dimensions"
-  [{:keys [count cpus mem] :as quota}
-   {:keys [count cpus mem] :as usage}]
+  [quota usage]
   (every? (fn [[usage-key usage-val]]
-            (<= usage-val (get quota usage-key 0)))
+            (<= usage-val (+ 0.01 (get quota usage-key 0))))
```

Comment:
Can we write a test for this (one that would fail without the added `0.01`)?

Reply:
Not really. I found this when I was trying to diagnose why some job's weren't launching in my GKE efforts. It ended up being unrelated to my problem, but I thought it was worth fixing.

Reply:
Why can't we write a test for it?

Reply:
We can contrive a set of jobs that add up and make it overflow. Given current tasking, the mildness of this bug (only important impact is potential on integration test flakiness), I don't want to sink more than minimal effort into this PR.

Reply:
Why does it only impact integration tests? Couldn't this affect real users?

Reply:
It could affect real users, but on the margin it only means you may get to launch one job less than you should. On discussion, we decided to defer the fix until we have time to write a unit test.